content/howto/geodesic/iam-creds-authentication.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Authenticate with AWS inside Geodesic"
+description: "Learn how to authenticate within Geodesic using AWS IAM Credentials."
+weight: 1
+draft: true
+---
+
+## Intro
+
+In this how-to, we'll provide a quick step-by-step guide on authenticating with AWS inside Geodesic.
+
+## Prerequisites
+
+### System Requirements
+
+You'll need to have an AWS Account with the ability to create credentials and have [Docker installed](https://docs.docker.com/get-docker/) on your local machine. **That's all**.
+
+### SweetOps Know-how
+
+We expect you've gone through the tutorial on ["Getting started with Geodesic"]({{< relref "tutorials/geodesic-getting-started.md" >}}) prior to this How-To since that contains some important understanding of what Geodesic is, how it works, and what it's doing.
+
+## How-To
+
+### 1. Install and start Geodesic
+
+First, at your terminal, let's install Geodesic!
+
+```bash
+docker run cloudposse/geodesic:latest-debian | bash
+```
+
+This will install Geodesic as a script at `/usr/local/bin/geodesic`, which you can then invoke like so:
+
+```bash
+geodesic
+```
+
+This small script is just a wrapper around running Geodesic via `docker` with a couple extra options (like mounting `$HOME` to `/localhost`). It should drop you into the Geodesic shell like you would expect.
+
+
+### 2. Authenticate with AWS + aws-vault
+
+Geodesic ships with [`aws-vault`]({{< relref "reference/tools.md#aws-vault" >}}) to help manage our credentials and retrieve access tokens from AWS to provide us with authenticated sessions. To set up a new profile, first [create a new IAM user and programmatic Access Key ID and Secret Key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console) and be sure to copy those values down somewhere. Now, in your Geodesic shell, let's do the following:
+
+```bash
+# Since our Geodesic shell is on linux, let's use the file backend which linux supports.
```

Comment:
```suggestion
# Since our Geodesic shell is on Linux, let's use the file backend which Linux supports.
# NOTE: It's not possible to use the OSX Keychain with `aws-vault` running inside of a Docker container.
```